osu.Game/Screens/Edit/Compose/Components/HitObjectInspector.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Extensions.TypeExtensions;
+using osu.Framework.Threading;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+
+namespace osu.Game.Screens.Edit.Compose.Components
+{
+    internal partial class HitObjectInspector : Screens.Edit.Compose.Components.EditorInspector
+    {
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            EditorBeatmap.SelectedHitObjects.CollectionChanged += (_, _) => updateInspectorText();
```

Comment:
I've intentionally kept this local for now. Can be refactored in the future after we figure out change tracking in the editor at a larger scale.